pyscf/ci/cisd.py

```diff
@@ -1004,7 +1007,9 @@ def get_init_guess(self, eris=None, nroots=1, diag=None):
             MP2 energy and initial guess(es) for CISD coefficients.
 
         '''
-        if eris is None: eris = self.ao2mo(self.mo_coeff)
+        if eris is None:
+            eris = self.ao2mo(self.mo_coeff)
+            self.e_hf = self.get_e_hf(mo_coeff=self.mo_coeff)
```

Comment:
self.e_hf may not be properly created if input includes eris

Reply:
I've deleted the self.e_hf evaluation.